tools/accuracy_checker/accuracy_checker/utils.py

```diff
@@ -296,9 +291,7 @@ def read_pickle(file: Union[str, Path], *args, **kwargs):
 
 def read_yaml(file: Union[str, Path], *args, ordered=True, **kwargs):
     with get_path(file).open() as content:
-        loader = orddict_loader or yaml.SafeLoader if ordered else yaml.SafeLoader
-        if not orddict_loader and ordered:
-            warn('yamlloader is not installed. YAML files order is not preserved. it can be sufficient for some cases')
+        loader = yaml.SafeLoader
         return yaml.load(content, *args, Loader=loader, **kwargs)
```

Comment:
```suggestion
        return yaml.safe_load(content, *args, **kwargs)
```

Reply:
I use directly SafeLoader, not Loader (common wrapper for all loaders) Loader.safe_load() === SafeLoader.load() according to docs